07week/calculator/index.html

```diff
@@ -0,0 +1,44 @@
+
```

Comment:
I see that you have defined all the functions, but they are not connected to your html file.